File: qlty-cli/tests/lang/java/basic.in/fields/GetterSetterDeclaration.java
package fields;

public class GetterSetterDeclaration {
  private String field;

  public String getFieldName() {
    return this.field;
  }

  public void setFieldName(String value) {
    this.field = value;
  }

  public static void main(String[] args) {
    GetterSetterDeclaration obj = new GetterSetterDeclaration();
    obj.setFieldName("Hello");
    System.out.println(obj.getFieldName());
  }
}

File: qlty-cli/tests/lang/csharp/basic.in/fields/GetterSetterDeclaration.cs
namespace Fields
{
    public class GetterSetterDeclaration
    {
        public string Field { get; set; }
        public string _field2

        public static void Main(string[] args)
        {
            GetterSetterDeclaration obj = new GetterSetterDeclaration();
            obj.Field = "Hello";
            obj._field2 = "World";
            System.Console.WriteLine(obj.Field);
        }
    }
}